assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -1105,9 +1104,37 @@ export default Component.extend({
   _onSendError(stagedId, error) {
     const stagedMessage = this.messageLookup[`staged-${stagedId}`];
     if (stagedMessage) {
-      stagedMessage.set("error", error.jqXHR.responseJSON.errors[0]);
-      this._resetAfterSend();
+      if (error.jqXHR?.responseJSON?.errors?.length) {
```

Comment:
I wanted to use `readyState` but this has proven to be super hard to fake in tests, and this is what we do in core.